client/app/vps/upgrade/vps-upgrade.html

```diff
@@ -0,0 +1,114 @@
+<section class="container-fluid cui-page__content cui-page__content-flex">
+
+    <div class="row d-lg-flex">
+        <div class="col-xm-12">
+
+            <cui-wizard-form data-form-title="{{ 'vps_dashboard_upgrade_vps' | translate }}"
+                             data-form-loaded="!$ctrl.loaders.init"
+                             data-form-on-cancel="$ctrl.resetAction()"
```

Comment:
You should not have a cancel on this particular form. Maybe a return button? We'll discuss it with CX.